app/Http/Requests/FilterTaskScheduleRequest.php

```
@@ -24,10 +25,9 @@ public function authorize(): bool
     public function rules(): array
     {
         return [
+            'timeline_classification' => ['required', Rule::in(TimelineClassification::values())],
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Validation\Rule' in method 'rules'. (undefined)

(StaticAccess)